### DISCUSSION TITLE
关于插件sync完成以后,正在使用时报错,stderr = "fatal: unable to access 'https://github.com/akinsho/

### DISCUSSION THREAD

**Discussion by yaotutu** [nshen/learn-neovim-lua#51] [Q&A] [answered]
* 源码是直接clone仓库的代码,仅做了几个按键的调整
* 我的网络环境是能访问github,但是不太稳定,所以插件更新的时候,偶尔会出错,但是重试以后提示安装完成,并且能看到编译后的文件.
* packer_compiled 这里的文件已经将插件的链接指向了本地地址,应该是直接访问本地就好.
* 是否是因为每次启动都需要自动更新,这时出现问题了?如果是的话,怎么禁止自动更新
![image](https://user-images.githubusercontent.com/21394924/176996679-e53a6ca4-1c4d-4843-96af-edf3f587ffff.png)
![image](https://user-images.githubusercontent.com/21394924/176996688-f1bb87e2-22ed-41da-bb59-5357b7794093.png)

**Comment by yaotutu** [comment]
https://github.com/yaotutu/nvim
这是我的配置文件

**Comment by nshen** [comment, marked as answer]
Hi,  简短回答，你需要把 `plugins.lua` 里这行删掉，重新更新一下就会更新到最新插件了，插件配置如果有问题可参考 v2 分支。

```
  -- 这里锁定插件版本在v1，不会继续更新插件
    snapshot = "v1",
```

详细原因是，我怕第三方插件更新太频繁，导致小册上配置和插件版本不匹配，所以在 main 分支上用 packer.nvim 的snapshots 锁定了插件版本：

https://raw.githubusercontent.com/nshen/learn-neovim-lua/main/snapshots/v1

我想可能是 snapshots 功能不太完善，每次你更新插件都会更新到插件的最新版本，然后使用时 packer.nvim 可能又会尝试 rollback 到我锁定的版本，但网络却不通导致报错。

我可能考虑要更新一下主分支，直接把commit id 写在安装那里，就不会有 rollback 报错了。

**Reply by yaotutu** [reply]
> Hi, 简短回答，你需要把 `plugins.lua` 里这行删掉，重新更新一下就会更新到最新插件了，插件配置如果有问题可参考 v2 分支。
> 
> ```
>   -- 这里锁定插件版本在v1，不会继续更新插件
>     snapshot = "v1",
> ```
> 
> 详细原因是，我怕第三方插件更新太频繁，导致小册上配置和插件版本不匹配，所以在 main 分支上用 packer.nvim 的snapshots 锁定了插件版本：
> 
> https://raw.githubusercontent.com/nshen/learn-neovim-lua/main/snapshots/v1
> 
> 我想可能是 snapshots 功能不太完善，每次你更新插件都会更新到插件的最新版本，然后使用时 packer.nvim 可能又会尝试 rollback 到我锁定的版本，但网络却不通导致报错。
> 
> 我可能考虑要更新一下主分支，直接把commit id 写在安装那里，就不会有 rollback 报错了。

问题已解决,感谢　🥳
